resources/cdk/sqs_lambda_to_batch_fargate/consumer_stack/consumer_stack.py

```diff
@@ -0,0 +1,202 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+from aws_cdk import (
+    aws_iam as iam,
+    aws_events as events,
+    aws_events_targets as targets,
+    aws_lambda as _lambda,
+    aws_lambda_event_sources as _event_sources,
+    aws_sqs as sqs,
+    aws_sns as sns,
+    aws_sns_subscriptions as subs,
+    aws_logs as logs,
+    aws_ec2 as ec2,
+    aws_ecs as ecs,
+    aws_ecr as ecr,
+    aws_batch as batch,
+    aws_batch_alpha as batch_alpha,
+    Stack
+)
+from constructs import Construct
+import json
+import os
+
+# Raises KeyError if environment variable doesn't exist
+language_name = os.environ["LANGUAGE_NAME"]
+
+class ConsumerStack(Stack):
+    def __init__(self, scope: Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+
+        #############################################
+        ##                                         ##
+        ##                 RESOURCES               ##
+        ##                                         ##
+        #############################################
+
+        # Define existing SNS topic in producer account
+        fanout_topic_name = "ProducerStack-fanouttopic6EFF7954-QUCVUVFOWZDA"
+        fanout_topic_arn = f'arn:aws:sns:us-east-1:808326389482:{fanout_topic_name}'
+        sns_topic = sns.Topic.from_topic_arn(self, fanout_topic_name, topic_arn=fanout_topic_arn)
+        container_image = ecs.EcrImage.from_registry(f"public.ecr.aws/b4v4v1s0/{language_name}:latest")
+
+        # client.get_parameter(Name=language_name, WithDecryption=True)
```

Comment:
Commented code. Either needs to be removed or an explanation given as to why it's comment/when to uncomment.